### ISSUE TITLE
Background bleeding through on individual screen images

### ISSUE THREAD

**Issue by mikeclayton** [mikeclayton/FancyMouse#44]
In the unit test code, some screen images have the background bleeding through around the top and right edges - e.g. this:

**preview image**

![image](https://github.com/user-attachments/assets/01f6b1c0-e16c-4dfd-99fc-bc1ec93d04b5)

**close-up**

The green background is bleeding through between the screen image and the screen bezel

![image](https://github.com/user-attachments/assets/c1bd3620-bf37-4363-95fa-710d92a1f247)

This is caused by sub-pixel smoothing when full-size screenshot images are stretched to fit the preview image. It only seems to be happening with the StaticImageRegionCopyService because that uses the Graphics class and GDI instead of Win32 StretchBlt.

See https://www.codeproject.com/Articles/14884/BorderBug for a discussion of this behaviour.

This can be fixed by drawing the preview screens with the following Graphics settings:

```
        previewGraphics.PixelOffsetMode = PixelOffsetMode.Half;
        previewGraphics.InterpolationMode = InterpolationMode.NearestNeighbor;
```

but this has to be done for *only* the DrawImage calls as all the bezel drawing will be offset to incorrect positions otherwise.

**Comment by mikeclayton** [comment]
After:

![image](https://github.com/user-attachments/assets/f22ae768-cb82-4054-a7d3-6bfc440126c3)

![image](https://github.com/user-attachments/assets/7a659133-9b11-4f86-a2d2-930c30cff388)

No background bleed.

**Comment by mikeclayton** [comment]
Fixed in v0.0.20-preview

https://github.com/mikeclayton/FancyMouse/releases/tag/v0.0.20-preview
